crates/api_models/src/payments.rs

```diff
@@ -2382,8 +2382,8 @@ pub struct PaymentRequestMetadata {
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
 pub struct SessionTokenInfo {
-    pub certificate: String,
-    pub certificate_keys: String,
+    pub certificate: Secret<String>,
```

Comment:
Add a `#[schema(value_type = String)]` annotation, this should fix the failing OpenAPI check.